app/main.py

```diff
@@ -1,8 +1,22 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
+
+    def __init__(self, name: str, age: int) -> None:
+        self.name = name
+        self.age = age
+        self.people[name] = self
 
 
 def create_person_list(people: list) -> list:
-    # write your code here
-    pass
+    person_list = [
+        Person(person["name"], person["age"])
+        for person in people
+    ]
+    persons = Person.people
+
+    for person in people:
+        if person.get("wife"):
+            persons[person["name"]].wife = persons[person["wife"]]
```

Comment:
You could use the walrus operator to avoid obtaining value from the dict twice, e.g.
```suggestion
        if wife_name := person.get("wife"):
            persons[person["name"]].wife = persons[wife_name]
```